Client.Core/Flux/Internal/FluxResultMapper.cs

```diff
@@ -16,6 +16,11 @@ namespace InfluxDB.Client.Core.Flux.Internal
 {
     internal class FluxResultMapper
```

Comment:
Please add a test into `FluxResultMapperTest.cs:` to test your changes. Something like:

```csharp
[Test]
public void MapTimestampDifferentName()
{
    var now = DateTime.UtcNow;
    
    var fluxRecord = new FluxRecord(0);
    fluxRecord.Values["tag"] = "production";
    fluxRecord.Values["min"] = 10.5;
    fluxRecord.Values["max"] = 20.0;
    fluxRecord.Values["avg"] = (Double) 18;
    fluxRecord.Values["_time"] = Instant.FromDateTimeUtc(now);
    
    var poco = _parser.ToPoco<PointWithoutTimestampName>(fluxRecord);
    
    Assert.AreEqual("production", poco.Tag);
    Assert.AreEqual(10.5, poco.Minimum);
    Assert.AreEqual(20, poco.Maximum);
    Assert.AreEqual(18, poco.Average);
    Assert.AreEqual(now, poco.Timestamp);
}

private class PointWithoutTimestampName
{
    [Column("tag", IsTag = true)] public string Tag { get; set; }
    [Column("min")] public double Minimum { get; set; }
    [Column("max")] public double Maximum { get; set; }
    [Column("avg")] public Double Average { get; set; }
    [Column(IsTimestamp = true)] public DateTime Timestamp { get; set; }
}
```